src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -35,6 +35,30 @@
             "linkType": "kfs"
           },
           {
+            "label": "Disbursement Voucher",
+            "link": "financialDisbursementVoucher.do?methodToCall=docHandler&command=initiate&docTypeName=DV",
+            "linkType": "kfs",
+            "permission": {
+              "templateNamespace": "KR-SYS",
+              "templateName": "Initiate Document",
+              "details": {
+                "documentTypeName": "DV"
+              }
+            }
+          },
+          {
+            "label": "Recurring Disbursement Voucher",
```

Comment:
The only question, probably a functional one, I have is whether Recurring Disbursement Voucher should be right after the Disbursement Voucher link, or later in the list in alphabetical order.